integrations/venice-duckdb/src/main/java/com/linkedin/venice/duckdb/DuckDBDaVinciRecordTransformerExample.java

```diff
@@ -0,0 +1,78 @@
+package com.linkedin.venice.duckdb;
+
+import static com.linkedin.venice.ConfigKeys.DA_VINCI_CURRENT_VERSION_BOOTSTRAPPING_SPEEDUP_ENABLED;
+
+import com.linkedin.avroutil1.compatibility.AvroCompatibilityHelper;
+import com.linkedin.d2.balancer.D2Client;
+import com.linkedin.d2.balancer.D2ClientBuilder;
+import com.linkedin.davinci.client.DaVinciClient;
+import com.linkedin.davinci.client.DaVinciConfig;
+import com.linkedin.davinci.client.DaVinciRecordTransformerConfig;
+import com.linkedin.davinci.client.factory.CachingDaVinciClientFactory;
+import com.linkedin.venice.D2.D2ClientUtils;
+import com.linkedin.venice.utils.PropertyBuilder;
+import com.linkedin.venice.utils.VeniceProperties;
+import io.tehuti.metrics.MetricsRepository;
+import java.nio.charset.StandardCharsets;
+import java.util.Collections;
+import java.util.Objects;
+import java.util.Set;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import org.apache.avro.Schema;
+import org.apache.avro.generic.GenericRecord;
+import org.apache.commons.io.IOUtils;
+
+
+public class DuckDBDaVinciRecordTransformerExample {
+  public static void main(String[] args) throws Exception {
+    DaVinciConfig clientConfig = new DaVinciConfig();
+    String clusterDiscoveryD2ServiceName = "venice-discovery";
+    MetricsRepository metricsRepository = new MetricsRepository();
+    String zkHosts = "zk-example-url.com:12913";
```

Comment:
Can we get this from the `args` rather than hardcoding? You can look at how the `AdminTool` leverages the `CommandLine` utility to handle CLI args parsing cleanly while minimizing boilerplate...